### PR TITLE
Enable multiple k-mer assembly steps using rnaSPAdes

### PR DESCRIPTION
**Pull request by mmokrejs**
Use odd numbers as k-mer sizes for the rnaSPAdes assembly.

Terminate the error message with a proper newline.

### CODE REVIEW

**Comment by mmokrejs** [comment]
Note both config.yaml files are modified in this pull request. If you want to kee trinity as the default please let me know.

**Comment by tomas-pluskal** [comment]
keep trinity as the default

File: Snakefile [hunk 1]
```diff
@@ -307,8 +307,17 @@ rule rnaspades:
     16
   shell:
     """
-    ##TODO = kmer shouldn't be fixed, & should be configurable from the beginning of the script (did run into some bugs with the auto parameter)
-    rnaspades.py --dataset {input.samples} -t {threads} -m {params.memory} -o rnaspades_out --only-assembler -k 47 &> {log}
+    myfiles=$(awk '{{print $3,$4}}' test_samples.txt | tr '\\n' ' ') 2>> {log}
```
Comment (tomas-pluskal) [inline Snakefile:310]: This code needs detailed comments to explain what is it actually supposed to do

Comment (mmokrejs) [reply Snakefile:396]: Done

Comment (tomas-pluskal) [reply Snakefile:396]: Not done! I mean detailed comments, explaining the purpose as well as the individual expressions.

File: Snakefile [hunk 2]
```diff
@@ -442,7 +456,12 @@ Runs Trinity script to get statistics about the assembled transcriptome
     1
   shell:
     """
-    {TRINITY_HOME}/util/TrinityStats.pl {input.transcriptome} > {output.stats} 2> {log}
+    assembler=$( grep '^assembler: ' config.yaml | sed -e 's/^assembler: "\(.*\)"/\1/' )
```
Comment (tomas-pluskal) [inline Snakefile:459]: No need to parse the config file manually here. See
https://stackoverflow.com/questions/49140578/how-can-one-access-snakemake-config-variables-inside-shell-section

**Comment by mmokrejs** [comment]
Sorry for the many push-forces, now it is hopefully in a single chunk with applied tweaks.

**Comment by tomas-pluskal** [comment]
Addressed with #55

**Pull request closed without merging**